### PR TITLE
[DOC] fix documentation issues #1027

### PR DESCRIPTION
**Pull request by Irallia**
Resolves all ticked parts of #1027 (#1424 is resolving two of these):

Resolves:
  - [x] Table in Alphabet Overview for Different concepts is too wide. (Plus, there are some weird boxes in column Aux)
  - [x] weird spacing in is_char_adaptation_v, is_uint_adaptation_v
  - [x] read_groups in alignment file in IO, barcode table is not correctly shown , same problem in ref_id_info

Resolved by #1424:
- [x] Table Type definition in simd too wide
- [x] Type deduction guides table in alignment file in  IO too wide

Additionally:
- [x] Scale image to textwidth in Pairwise Alignment

(cherry picked from commit 0fdbfc142daaa17da7a94bfacb6e7dd4e7464516)

### CODE REVIEW

File: include/seqan3/alphabet/adaptation/char.hpp [hunk 1]
```diff
@@ -33,10 +33,8 @@ namespace seqan3::detail
 //!\brief Whether a type is `char`, `char16_t`, `char32_t` or `wchar_t` (type trait).
 //!\ingroup adaptation
 template <typename type>
-constexpr bool is_char_adaptation_v = std::same_as<type, char>     ||
-                                      std::same_as<type, char16_t> ||
-                                      std::same_as<type, char32_t> ||
-                                      std::same_as<type, wchar_t>;
+constexpr bool is_char_adaptation_v = std::same_as<type, char> || std::same_as<type, char16_t> ||
+    std::same_as<type, char32_t> || std::same_as<type, wchar_t>;
```
Comment (marehr) [inline include/seqan3/alphabet/adaptation/char.hpp]: this change seems unrelated

File: include/seqan3/alphabet/adaptation/uint.hpp [hunk 2]
```diff
@@ -33,9 +33,8 @@ namespace seqan3::detail
 //!\brief Whether a type is `uint8_t`, `uint16_t` or `uint32_t`.
 //!\ingroup adaptation
 template <typename type>
-constexpr bool is_uint_adaptation_v = std::same_as<type, uint8_t>  ||
-                                      std::same_as<type, uint16_t> ||
-                                      std::same_as<type, uint32_t>;
+constexpr bool is_uint_adaptation_v = std::same_as<type, uint8_t> || std::same_as<type, uint16_t> ||
+    std::same_as<type, uint32_t>;
```
Comment (marehr) [inline include/seqan3/alphabet/adaptation/uint.hpp]: this change seems unrelated

File: include/seqan3/io/alignment_file/header.hpp [hunk 3]
```diff
@@ -140,23 +140,12 @@ class alignment_file_header
...
-     *         alternative, in the format 'chr:start-end', 'chr' (if known),
-     *         or '*' (if unknown), where 'chr' is a sequence in the primary assembly.
-     *         Must not be present on sequences in the primary assembly. |
-     * | AN  | Alternative reference sequence names. A comma-separated list
-     *         of alternative names that tools may use when referring to this
-     *         reference sequence. These alternative names are not used elsewhere
-     *         within the SAM file; in  particular, they must not appear in alignment
-     *         records’ RNAME or RNEXT fields. regular expression : name (, name )*
-     *         where name is [0-9A-Za-z][0-9A-Za-z*+.@ |-]* |
+     * | AH  | Indicates that this sequence is an alternate locus. The value is the locus in the primary assembly for which this sequence is an alternative, in the format 'chr:start-end', 'chr' (if known), or '*' (if unknown), where 'chr' is a sequence in the primary assembly. Must not be present on sequences in the primary assembly. |
```
Comment (marehr) [inline include/seqan3/io/alignment_file/header.hpp]: Is there a reason for these changes? They don't fit our style-guide of at most 120 chars per line.

**Comment by marehr** [comment]
@Irallia Can you give a bit more detail which checkmarks of #1027 this PR resolve?

File: include/seqan3/alphabet/adaptation/char.hpp (hunk 1 above)
Comment (Irallia) [reply include/seqan3/alphabet/adaptation/char.hpp]: Resolves: "weird spacing in is_char_adaptation_v, is_uint_adaptation_v"

File: include/seqan3/alphabet/adaptation/uint.hpp (hunk 2 above)
Comment (Irallia) [reply include/seqan3/alphabet/adaptation/uint.hpp]: Resolves: "weird spacing in is_char_adaptation_v, is_uint_adaptation_v"

File: include/seqan3/io/alignment_file/header.hpp (hunk 3 above)
Comment (Irallia) [reply include/seqan3/io/alignment_file/header.hpp]: Resolves: "read_groups in alignment file in IO, barcode table is not correctly shown , same problem in ref_id_info"

**Comment by marehr** [comment]
Thank you for answering. I put this PR as discussion for the next core meeting (unfortunately next year), because I'm not sure how to proceed here. I don't think that we want to break our code style for documentation fixes.

/cc: @smehringer, @eseiler, @rrahn

File: include/seqan3/alphabet/adaptation/char.hpp (hunk 1 above)
Comment (Irallia) [reply include/seqan3/alphabet/adaptation/char.hpp]: hideinitializer

File: include/seqan3/alphabet/adaptation/uint.hpp (hunk 2 above)
Comment (Irallia) [reply include/seqan3/alphabet/adaptation/uint.hpp]: same here

File: include/seqan3/io/alignment_file/header.hpp (hunk 3 above)
Comment (Irallia) [reply include/seqan3/io/alignment_file/header.hpp]: list 

* AH sdafsaaffa
* AN sdasdsa

File: include/seqan3/alphabet/all.hpp [hunk 4]
```diff
@@ -123,7 +123,7 @@
  * seqan3::writable_alphabet whenever the values might be changed.
  * Semi-alphabets are less useful in application code.
  *
- * |                                  | seqan3::semialphabet | seqan3::writable_semialphabet | seqan3::alphabet | seqan3::writable_alphabet | Aux |
+ * |                                  | [semialphabet](seqan3::semialphabet) | [writable_semialphabet](seqan3::writable_semialphabet) | [alphabet](seqan3::alphabet) | [writable_alphabet](seqan3::writable_alphabet) | Aux |
```
Comment (rrahn) [inline include/seqan3/alphabet/all.hpp]: why was the link not preserved with the seqan3:: namespace?

Comment (joergi-w) [reply include/seqan3/alphabet/all.hpp]: The link was preserved, but the table was too wide, so it reached out of the display on the right. In this way there is just the important information shown and still linked.

Comment (rrahn) [reply include/seqan3/alphabet/all.hpp]: I know that the link is aliased and still works.
What I mean is that the only reason was to limit the size of the table, then I am in favor of making the same for the links in the row. Otherwise there is an inconsistency. I'd like to have the inconsistency at least consistent. Meaning that all tables don't use `seqan3::` prefix but everything else does.

**Comment by h-2** [comment]
@seqan/core @Irallia 
This PR made the tables in http://docs.seqan.de/seqan/3-master-user/group__alphabet.html look even worse than before :zipper_mouth_face:
